crates/web/frontend/next.config.mjs

```diff
@@ -1,4 +1,15 @@
+const host = process.env.API_HOST || "localhost";
```

Comment:
I think we should join all those variables into a single one. and also, the scheme http/https, because maybe the api is outside of the host and we have to access through https

```suggestion
const api_endpoint = process.env.API_ENDPOINT || "https://localhost:3001";
```

`API_ENDPOINT` or `API_URL`

I think it is better to simplify the number of env vars, because with the current way to do it, adding the protocol schema would mean another env var.